toolkit/src/components/cut/list-item/service-terminating-gateway/index.hbs

```diff
@@ -0,0 +1,64 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<Cut::ListItem
+  @href={{@href}}
+  @isRouteExternal={{@isRouteExternal}}
+  @route={{@route}}
+  @isHrefExternal={{@isHrefExternal}}
+  @query={{@query}}
+  @replace={{@replace}}
+  @onClick={{@onClick}}
+  data-test-service-terminating-gateway-list-item
+  as |L|>
+  <L.Content as |C|>
+    <Cut::ListItem::Template as |T|>
+      <T.Label as |LB|>
+        <LB.Title data-test-service-name>{{@service.name}}</LB.Title>
+        <LB.Metadata>
+          {{#if this.isAllHealthy}}
+            <Hds::Badge
+              @color='success'
+              @icon='check'
+              @text='Healthy'
+              @size="small"
+              data-test-health-check-all-healthy
+            />
+          {{else}}
+            <!--Instance health check-->
+            {{#if @service.metadata.healthCheck.instance}}
+              <Cut::Metadata::HealthCheckBadgeSet
+                @successCount={{@service.metadata.healthCheck.instance.success}}
+                @warningCount={{@service.metadata.healthCheck.instance.warning}}
+                @criticalCount={{@service.metadata.healthCheck.instance.critical}}
+                @type="instance"
+              />
+            {{/if}}
+          {{/if}}
+          {{#if @service.metadata.kindName}}
+            <Cut::TextWithIcon @icon="gateway" @text={{@service.metadata.kindName}} data-test-service-kind />
+          {{/if}}
+
+          {{#if (not-eq @service.metadata.linkedServiceCount undefined)}}
+            <!-- TODO: add translate with pluralize (CC-5314) -->
+            <span data-test-associated-service-count>{{@service.metadata.linkedServiceCount}} linked services</span>
```

Comment:
Correct me if I am wrong, but it seems like the only difference between the two gateways is that this and the `# upstreams` bit?

Reply:
yea, as of now. Let's have one service list item for all of them. Before the review and checking with Jake there were much more differences between gateways and service 

I think we will just check what's passed and based on that will present data.